### PR TITLE
updated header text size on landing, about and team. Replaced about h…

### PR DESCRIPTION
**Pull request by Letsgohyde**
1. Decreased text size on landing, about and team pages. 
2. Replaced hero image on about page
3. Added media query to reduce text size and image on mobile view. 
4. Added margin and padding to allow text to breath.

### CODE REVIEW

File: themes/delphi/assets/css/pages/_team.scss [hunk 1]
```diff
@@ -6,3 +6,17 @@
 .team-member > img {
     border-radius: 5px;
 }
+
+#team-header {
+    line-height: 64px;
+    font-size: 40;
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_team.scss:12]: please add a unit, e.g. pixel

File: themes/delphi/assets/css/pages/_team.scss [hunk 2]
```diff
@@ -6,3 +6,17 @@
 .team-member > img {
     border-radius: 5px;
 }
+
+#team-header {
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_team.scss:10]: is possible prefer a CSS class over a unique ID

File: themes/delphi/layouts/_default/about.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {{ define "main" }}
 
 <div class="about-mission uk-cover-container">
-    <img class="about-mission-img" src="{{ "images/mission.jpg" | relURL }}" alt="Mission image" uk-cover />
+    <img class="about-mission-img" src="{{ "images/red-bubble-map_mission-pg.jpg" | relURL }}" alt="Mission image" uk-cover />
```
Comment (sgratzl) [inline themes/delphi/layouts/_default/about.html:4]: please delete the old image

File: themes/delphi/assets/css/pages/_team.scss (hunk 1 above)
Comment (Letsgohyde) [reply themes/delphi/assets/css/pages/_team.scss]: Thanks, changes made :)

File: themes/delphi/assets/css/pages/_team.scss (hunk 2 above)
Comment (Letsgohyde) [reply themes/delphi/assets/css/pages/_team.scss]: updated to class, thanks

File: themes/delphi/layouts/_default/about.html (hunk 3 above)
Comment (Letsgohyde) [reply themes/delphi/layouts/_default/about.html:4]: removed, thanks :)

**Comment by Letsgohyde** [comment]
I've made the recommended changes, thanks Sam!

**Review by sgratzl — APPROVED** [review]
👍 code wise

**Comment by Letsgohyde** [comment]
@tildechris let me know what you think of the updates to the About page :) thanks!

**Comment by tildechris** [comment]
@Letsgohyde this looks great.  I made a change to the responsive layout and height of the image and merged in from dev because you also changed the collaborators section.

File: themes/delphi/assets/css/pages/_about.scss [hunk 4]
```diff
@@ -1,48 +1,59 @@
-
 .about-mission {
-    padding: 8rem 0;
+    background-image: url("../images/red-bubble-map_mission-pg-short.jpg");
+    background-size: auto;
+    height: 200px;
 }
 
 .about-mission-img {
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_about.scss:7]: seems unused

File: themes/delphi/assets/css/pages/_about.scss [hunk 5]
```diff
@@ -1,48 +1,59 @@
...
 }
-
-@media (min-width: $breakpoint-medium) {
+@media screen and (min-width: $breakpoint-medium) {
     .about-description {
         background: white;
         border: 1px solid #D3D4D8;
         border-radius: 5px;
+    }
+    .about-description {
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_about.scss:42]: isn't is the same selector and thus can be merged?

File: themes/delphi/layouts/_default/about.html [hunk 6]
```diff
@@ -1,23 +1,13 @@
...
-        <h1 class="grid-2-10 about-mission-text">
-            {{.Site.Params.mission}}
-        </h1>
-    </div>
+<div class="about-mission">
 </div>
-
 <div class="uk-container content-grid">
     <div class="grid-2-10 about-description">
+        <div>
```
Comment (sgratzl) [inline themes/delphi/layouts/_default/about.html:7]: do we need the wrapper div?

File: themes/delphi/layouts/_default/about.html [hunk 7]
```diff
@@ -1,23 +1,13 @@
...
-    <div class="uk-container content-grid">
-        <h1 class="grid-2-10 about-mission-text">
-            {{.Site.Params.mission}}
-        </h1>
-    </div>
+<div class="about-mission">
 </div>
-
 <div class="uk-container content-grid">
     <div class="grid-2-10 about-description">
```
Comment (sgratzl) [inline themes/delphi/layouts/_default/about.html:6]: doesn't seem to be fully centered in the grid:

image

File: themes/delphi/assets/css/pages/_team.scss [hunk 8]
```diff
@@ -3,6 +3,21 @@
...
 }
+
+.team-header {
+    line-height: 64px;
+    font-size: 40px;
+    padding-top: .25em;
+    padding-bottom: 1em;
+}
+
+@media screen and (max-width: 414px) {
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_team.scss:18]: should is also be the $breakpoint-medium or $breakpoint-small?

File: themes/delphi/assets/css/pages/_landing.scss [hunk 9]
```diff
@@ -74,12 +74,22 @@ $carousel-height:500px;
...
     p {
-        font-weight: 600;
-        font-size: 32px;
+        font-weight: 500;
+        font-size: 28px;
+        line-height: 48px;
+        color: rgb(35, 39, 53);
     }
+
+    @media screen and (max-width: 414px) {
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_landing.scss:87]: should is also be the $breakpoint-medium or $breakpoint-small?

File: themes/delphi/assets/css/pages/_about.scss [hunk 10]
```diff
@@ -1,48 +1,59 @@
...
+    height: 200px;
 }
 
 .about-mission-img {
-    z-index: -1;    
+    z-index: -1;
+    filter: grayscale(65%);  
 }
+
 .about-mission-mission {
```
Comment (sgratzl) [inline themes/delphi/assets/css/pages/_about.scss:12]: seems unused

File: themes/delphi/layouts/_default/about.html (hunk 7 above)
Comment (tildechris) [reply themes/delphi/layouts/_default/about.html:6]: This seems to have been an conscious design decision from Raph.  Let's defer.

**Comment by tildechris** [comment]
Sam, I addressed your feedback items.  Please, take a look.

**Comment by tildechris** [comment]
Merging in order to make release branch.

**Pull request merged**